Capgemini.Xrm.CdsDataMigrator/Capgemini.Xrm.CdsDataMigratorLibrary/Forms/ExportLookupMappingsForm.cs

```diff
@@ -88,6 +89,22 @@ public List<DataGridViewRow> Mappings
 
         #endregion
 
+        #region action mappings
+
+        [ExcludeFromCodeCoverage]
+        public void SetMapFieldToNull()
+        {
+            dgvMappings.Rows[dgvMappings.CurrentCell.RowIndex].Cells[2].Value = null;
+        }
+
+        [ExcludeFromCodeCoverage]
+        private void SetCurrentCellIsUpdated(bool isUpdated)
+        {
+            CurrentCellIsUpdated = isUpdated;
+        }
```

Comment:
Is this really needed? Is it just to obtain 100% code coverage?

Reply:
Yeah it's unnecessary. I've removed the method and am updating CurrentCellIsUpdated directly now